### ISSUE TITLE
$viewsPath resolves to same path regardless path existence

### ISSUE THREAD

**Issue by mnishihan** [jeroennoten/Laravel-AdminLTE#417]
https://github.com/jeroennoten/Laravel-AdminLTE/blob/2bcfec66c233857f675b4607f8e4302d4a5b0065/src/AdminLteServiceProvider.php#L50-L56

Couldn't this entire function be defined as below, given that `$this->packagePath('resources/views')` always exist?

```php
private function loadViews()
{
       $viewsPath = $this->packagePath('resources/views');
       $this->loadViewsFrom($viewsPath, 'adminlte');
}
```

**Comment by REJack** [comment]
Then it's not possible use the published vendor files to modify them instead of touching the package files

**Comment by REJack** [comment]
I see there is failure in the code 🤦‍♂ it should first use the base resource path instead of package resource path

**Comment by mnishihan** [comment]
> Then it's not possible use the published vendor files to modify them instead of touching the package files

That's not true. [loadViewsFrom()](https://github.com/laravel/framework/blob/5e10c01d31356296dfb16b95017f0e1be466f660/src/Illuminate/Support/ServiceProvider.php#L88)` in `Illuminate\Support\ServiceProvider` already takes care of published views in `resources/views/vendor/$namespace` folder if found, otherwise fallbacks to package provided views.

**Comment by REJack** [comment]
Oh I see what you mean, there is already a fallback in the ServiceProvider, then its totally unnecessary. I'll add a PR with a correction 😄

**Comment by REJack** [comment]
I've fixed it in PR #421.

**Comment by lacodimizer** [comment]
merged, published in v3.0.3
